public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplate.utils.ts

```diff
@@ -13,3 +13,12 @@ export const formatTemplate = (template: Template): FormattedTemplate => {
 };
 
 export const formatTemplates = (templates: Template[]): FormattedTemplate[] => templates.map(formatTemplate);
+
+export const beautifyUnit = (unit: TemplateParamUnit) => {
+  const unitMap: Record<TemplateParamUnit, string> = {
```

Comment:
Can this go to constants file or be moved out of the function? So it doesn't get created everytime the function is called since this is constant.

Reply:
Sure can 👍